frontend/src/components/forms/CreateResident.tsx

```diff
@@ -0,0 +1,254 @@
+import React, { useEffect, useState } from "react";
+import Select, { SingleValue } from "react-select";
+import {
+    Alert,
+    AlertDescription,
+    AlertIcon,
+    Box,
+    Button,
+    FormControl,
+    FormErrorMessage,
+    FormLabel,
+    Input,
+    Modal,
+    ModalBody,
+    ModalContent,
+    ModalHeader,
+    ModalCloseButton,
+    ModalOverlay,
+    ModalFooter,
+    ScaleFade,
+    Divider,
+} from "@chakra-ui/react";
+
+import { AddIcon } from "@chakra-ui/icons";
+import { SingleDatepicker } from "chakra-dayzed-datepicker";
+import { Card, Col, Row } from "react-bootstrap";
+import CommonAPIClient from "../../APIClients/CommonAPIClient";
+
+import selectStyle from "../../theme/forms/selectStyles";
+import { singleDatePickerStyle } from "../../theme/forms/datePickerStyles";
+
+// TODO: Connect to Buidings table 
+const BUILDINGS = [
+    { label: "144", value: "144" },
+    { label: "362", value: "362" },
+    { label: "402", value: "402" },
+];
+
+const CreateResident = (): React.ReactElement => {
+
+    const [initials, setInitials] = useState("");
+    const [roomNumber, setRoomNumber] = useState("");
+    const [moveInDate, setMoveInDate] = useState(new Date());
+    const [building, setBuilding] = useState("");
+
+    const [initialsError, setInitialsError] = useState(false);
+    const [roomNumberError, setRoomNumberError] = useState(false);
+    const [moveInDateError, setMoveInDateError] = useState(false);
+    const [buildingError, setBuildingError] = useState(false);
+
+    const [isOpen, setIsOpen] = useState(false);
+    const [showAlert, setShowAlert] = useState(false);
+
+    const addResident = async () => {
+        await CommonAPIClient.createResident({
+            initial: initials.toLowerCase(),
```

Comment:
convert it to upper case:
```Suggestion
            initial: initials.toUpperCase(),
```

Reply:
I thought the initials are stored as lowercase in the database, is switching it to uppercase going to cause issues?

![image](https://github.com/uwblueprint/supportive-housing/assets/43042601/71513fd1-c3bf-4d30-bf02-4c232c6de621)

Reply:
its not going to cause issues, we just have to cast it to upper case when we render it on the front-end, so its easier to just store it as upper case